VideoWeb/VideoWeb/ClientApp/src/app/shared/venue-list/venue-list.component.html

```diff
@@ -2,30 +2,36 @@
   <div class="govuk-grid-column-full">
     <h1 class="govuk-heading-m">{{'venue-list.title' | translate}}</h1>
   </div>
+
     <div class="govuk-grid-column-three-quarters">
-        <ng-select id="venue-allocation-list" class="selection-list"
+        <label>
```

Comment:
An empty label won't fix accessibility. Ng-Select has an attribute `labelForId`. Here's a snippet of an example from the library repo. [Full example here](src/demo/app/examples/forms-async-data-example/forms-async-data-example.component.html)

```
<label for="album">Loading async data</label>
<ng-select [items]="albums"
           #select
           bindLabel="title"
           dropdownPosition="auto"
           bindValue="id"
           labelForId="album"
           placeholder="Select album"
           [virtualScroll]="true"
           formControlName="album">
    <ng-template ng-option-tmp let-item="item" let-search="searchTerm">
        <div><span>Title: </span><span [ngOptionHighlight]="search">{{item.title}}</span></div>
        <small><b>Id:</b> {{item.id}} | <b>UserId:</b> {{item.userId}}</small>
    </ng-template>
</ng-select>
```

This applies to the second ng-select instance

Reply:
the empty label is fixing the Axe. Your fix will not work as I have already tried it. Please run the Axe tool and you will see it.